helm/charts/infra/templates/connector/clusterrole.yaml

```diff
@@ -36,4 +36,61 @@ rules:
       - watch
       - bind
       - escalate
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: pod-logs
+  labels:
+{{- include "connector.clusterRoleLabels" . | nindent 4 }}
+rules:
+  - apiGroups: [""]
+    resources:
+      - pods
```

Comment:
This, and both cluster roles below, are pretty restrictive. `kubectl logs` also takes deployment, replica set, stateful set, daemon set, as arguments which this cluster role does not permit. To allow those use cases, we'll need to add `deployments`, `replicasets`, etc. to the cluster role resources

Reply:
This is super cool. To enable users have `kubectl logs [deployment|rs|pod]` under one `logs` role, let's just do that here. Is there a downside to that?

Reply:
Not really. It gives more permissions but covers a broader set of use cases